src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.ts

```diff
@@ -186,6 +187,15 @@ export class FileUploadSubmissionComponent implements OnInit, ComponentCanDeacti
         }
     }
 
+    /**
+     * Find "Unreferenced Feedback" item for Result, if it exists.
+     */
+    get unreferencedFeedback(): Feedback[] | undefined {
+        if (this.result && this.result.feedbacks && Array.isArray(this.result.feedbacks)) {
```

Comment:
Why are you using `Array.isArray(this.result.feedback)` should be an array or undefined anyways or not?

Reply:
This is also for the other exercise types.

Reply:
I just reused this method. Don't know if the developing person had something particular in mind, but since feedback can indeed only be an array or null this check could probably be removed